### DISCUSSION TITLE
For Woocommerce Shop Page Only

### DISCUSSION THREAD

**Discussion by SharatJaswal** [addonify/addonify-floating-cart#175] [Q&A] [answered]
Hi

This is, hands down, the best floating cart plugin. However if you can help with two queries, it will be highly appreciable.

One, how can I showcase this cart, only on the woocommerce shop page?  is_woocommerce() or is_shop() conditional tag. Where can I add that?

Two, how can I disable showcasing quantities of the item in the cart? I have disabled quantity in woocommerce settings, but the cart shows 1 alongside the item. The + and - sign don't work, as per the settings which is fine, but If I can disable the view altogether, that would be perfect.

Thanks a lot for this beautiful plugin. Makes the website stand out from the rest.

**Comment by h1dd3nsn1p3r** [comment, marked as answer]
Hi @SharatJaswal 

Thank you for the kind feedback. 

Please allow me to address your queries. 

1. We plan to add the conditional option on the setting page in future updates. However, we can accomplish that requirement with just a few lines of CSS code and let me share that with you. 

Copy the below CSS code & paste that into Dashboard > Appearances> Customizer> Additional CSS box.

```
#adfy__woofc-trigger {

    display: none; /* Hide globally */
}

body.woocommerce-page #adfy__woofc-trigger {

    display: flex; /* Show for shop pages */
}
```

2. Allow me some time to check this issue. But, if you wish to hide all the quality input (+ and -) buttons inside cart you can do that with this CSS code below. 

```
#adfy__floating-cart .adfy__woofc-content .adfy__woofc-content-entry .adfy__woofc-item .adfy__woofc-item-content .adfy__woofc-quantity {

    display: none; /* Hide quantity input */
} 
```

This is how the additional CSS box will look once you paste all the CSS code.

![image](https://user-images.githubusercontent.com/21140404/236979969-fc8621c3-7d5a-4dca-8f99-d1ce24f7c75d.png) 

Looking forward :)

**Reply by SharatJaswal** [reply]
This works like a charm. Very elegant, and looks perfect.
Can't thank you enough, Anuj!

**Reply by SharatJaswal** [reply]
In future updates, I hope you can integrate the "Empty Cart" button option along with the options of "Close Cart" and "Open Cart" button next to Checkout.

<s>And perhaps an option of offsetting the "Floating Cart" icon in design elements. For example, I would prefer the cart icon in the middle right of the page but we only have an option for top right/left or bottom right/left.</s>
Nevermind, I just needed coffee. Found it.

Again, thank you so much for developing this plugin. Wish you great success with it :)

**Reply by h1dd3nsn1p3r** [reply]
👋 Hi,

1. We recently added an option to choose the button check this thread https://github.com/addonify/addonify-floating-cart/issues/158. 
2. Regarding the Floating Cart icon yes, we can have that in upcoming releases. I have created a new thread here https://github.com/addonify/addonify-floating-cart/issues/176. 

I hope you found Addonify Floating Cart useful! Could you please leave us a quick review [here](https://wordpress.org/support/plugin/addonify-floating-cart/reviews/). It will help us to grow and stay motivated to deliver more features and improvements.

**Reply by SharatJaswal** [reply]
I left a review. You deserve it :)
